lib/index.js

```diff
@@ -21,6 +24,9 @@ function convertToAppleNews (articleJson, opts) {
     identifier: opts.identifier,
     title: articleJson.title,
     language: articleJson.language || 'en',
+    dateModified: formatDate(now),
```

Comment:
Shouldn't this be an option? Like articleJson.modifiedDate - we have this date in out db fwiw